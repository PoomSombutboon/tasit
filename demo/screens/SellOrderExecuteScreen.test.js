import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import SellOrderExecuteScreen from "./SellOrderExecuteScreen";

describe("SellOrderExecuteScreen", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    const sellOrder = {
      id: 123,
      priceMana: 0,
      priceUSD: 0,
      asset: {
        id: -1,
        name: "Not found",
        img: "https://decentraland.org/images/logo-65f7b27caf.png",
      },
    };
    const navigation = { getParam: () => sellOrder };
    expect(
      shallow(<SellOrderExecuteScreen navigation={navigation} />)
    ).toMatchSnapshot();
  });
});
