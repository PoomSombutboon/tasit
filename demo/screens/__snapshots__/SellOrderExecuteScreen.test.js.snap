// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellOrderExecuteScreen renders the component 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
